pages/v/[id]/embed.tsx

```diff
@@ -18,16 +19,25 @@ type Params = {
 export type Props = {
   playbackId: string,
   shareUrl: string,
-  poster: string
+  poster: string,
+  width: number,
+  height: number,
+  aspectRatio: number,
+  videoExists: boolean,
 };
 
 export const getStaticProps: GetStaticProps = async (context)  => {
   const { params } = context;
   const { id: playbackId } = (params as Params);
```

Comment:
nit (non-blocking): there's a lot of duplicate code hereand in `player-page-utils`. Consider abstracting/sharing.